src/Microsoft.SqlTools.ServiceLayer/SqlProjects/Contracts/Projects/NewSqlProject.cs

```diff
@@ -0,0 +1,33 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlServer.Dac.Projects;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+
+namespace Microsoft.SqlTools.ServiceLayer.SqlProjects.Contracts
+{
+    /// <summary>
+    /// Parameters for creating a new SQL Project
+    /// </summary>
+    public class NewSqlProjectParams : SqlProjectParams
```

Comment:
should we also have an option to set the Microsoft.Build.Sql version for new SDK-style projects?

Reply:
Good idea, though we'd need to add support for that down in DacFx since that's currently hardcoded.

Reply:
PR out for DacFx; will hook this up once that's checked in and nugeted.